### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by joxeru** [upscayl/upscayl#892] [Q&A]
Hi, can I use upscayl for commercial use?
I have downloaded it and it is amazing, but I would like to know if I can upscale some of my photos and then sell them in microstock sites, such as shutterstock.
The first thing is license - can I use for commercial use? And next thing - it is AI, so is it ok to use my upscaled images still as my work?
thanks for any help

**Comment by NayamAmarshe** [comment]
Hi, Upscayl uses the AGPL v3 license.
Some models are not fit for commercial usage. This is what we're trying to fix with Upscayl Cloud by offering commercial models.
